src/store/audio.ts

```diff
@@ -260,8 +260,11 @@ export const audioStore = createPartialStore<AudioStoreTypes>({
 
   SELECTED_AUDIO_KEYS: {
     getter(state) {
-      const base = new Set(state._selectedAudioKeys || []);
-      return Array.from(base);
+      return (
+        state._selectedAudioKeys?.filter((audioKey) =>
+          state.audioKeys.includes(audioKey)
+        ) || []
+      );
```

Comment:
これもわからなくなりそうなので理由書いときますか！
```suggestion
      return (
        //  undo/redoで消えていることがあるためフィルタする
        state._selectedAudioKeys?.filter((audioKey) =>
          state.audioKeys.includes(audioKey)
        ) || []
      );
```